modules/tinymce/src/core/main/ts/content/TransparentElements.ts

```diff
@@ -7,7 +7,9 @@ import * as NodeType from '../dom/NodeType';
 
 export const transparentBlockAttr = 'data-mce-block';
 
-const makeSelectorFromSchemaMap = (map: SchemaMap) => Arr.filter(Obj.keys(map), (key) => /^[a-z]+$/.test(key)).join(',');
+export const elementNames = (map: SchemaMap): string[] => Arr.filter(Obj.keys(map), (key) => !/[A-Z]/.test(key));
```

Comment:
Could just be me but I initially found it a bit hard to reason about why this works instead of the old way. I understand there is a test to ensure this doesn't get changed and regress which is the main thing. I am just wondering if it could be worth having a comment that mentions what the function is doing and how.

Reply:
Added a comment hopefully it's makes it easier to understand.